examples/twobee/twobee/app.py

```diff
@@ -0,0 +1,38 @@
+import random
+import toga
+from toga.style.pack import Pack, COLUMN, CENTER
+
+HEADS = toga.Image('https://beeware.org/static/images/brutus-270.png')
+TAILS = toga.Image('https://beeware.org/project/projects/libraries/toga/toga.png')
+TOSS = toga.Image('https://beeware.org/project/projects/tools/cricket/cricket.png')
```

Comment:
Ideally these would be resources packaged with the app, rather than resources downloaded from the web (unless you're looking to demonstrate images can be network resources?)

Reply:
I tried that but ran into a bug where app_dir wasn't defined.